smithy-linters/src/test/resources/software/amazon/smithy/linters/errorfiles/http-api-key-scheme-trait-validator-test.json

```diff
@@ -0,0 +1,39 @@
+{
+    "smithy": "1.0",
+    "shapes": {
+        "ns.foo#MyService": {
+            "type": "service",
+            "version": "2017-01-17",
+            "operations": [
+                {
+                    "target": "ns.foo#A"
+                }
+            ],
+            "traits": {
+                "smithy.api#httpApiKeyAuth": {
+                    "scheme": "Baz",
+                    "name": "ApiKeyName",
+                    "in": "query"
+                }
```

Comment:
You should add a positive case in here as well.

Unrelated, you can write these in the IDL too which is quite a bit easier